src/external/vtkio.jl

```diff
@@ -5,49 +5,55 @@ import WriteVTK: vtk_grid, vtk_save
 
 The function takes in the VTK filename and the scfres structure and stores into a VTK file.
 
-Grid Values:
+Parameters
+- `eigenvectors`: Store the eigenvectors or not. By default they are not stored.
```

Comment:
we don't call them eigenvectors anywhere else do we?

Reply:
Yes whatever.